Crash when crash reporting is enabled

At least on Debian 12, the Domain server will break if you enable crash reporting via the web UI.
I only checked this on 2023.11.1.

GCC actually throws a warning when building CrashHandlerBackend_Crashpad.cpp:
```gcc
2023-12-31T17:51:31.4930334Z [ 26%] Building CXX object libraries/networking/CMakeFiles/networking.dir/src/crash-handler/CrashHandlerBackend_Crashpad.cpp.o
2023-12-31T17:51:31.9781439Z In file included from /__w/overte/overte/libraries/networking/src/crash-handler/CrashHandlerBackend_Crashpad.cpp:37:
2023-12-31T17:51:31.9785317Z /__w/overte/overte/build/ext/makefiles/crashpad/project/src/crashpad/include/client/crashpad_info.h:223:18: warning: multi-character character constant [-Wmultichar]
2023-12-31T17:51:31.9790488Z   223 |     kSignature = 'CPad',
2023-12-31T17:51:31.9791636Z       |                  ^~~~~~
2023-12-31T17:51:33.3441900Z [ 26%] Building CXX object libraries/gpu/CMakeFiles/gpu.dir/src/gpu/Shader.cpp.o
```
Though this appears to be an issue within crashpad itself.

--- a/libraries/networking/src/crash-handler/CrashHandlerBackend_Crashpad.cpp
+++ b/libraries/networking/src/crash-handler/CrashHandlerBackend_Crashpad.cpp
@@ -457,6 +457,11 @@ bool startCrashHandler(std::string appPath, std::string crashURL, std::string cr
 }
 
 void setCrashReportingEnabled(bool enabled) {
+    if (!crashpadDatabase) {
+        qCCritical(crash_handler) << "Can't set to enabled, crash handler not initialized!";
+        return;
+    }
+
     auto settings = crashpadDatabase->GetSettings();
     settings->SetUploadsEnabled(enabled);
 

